backend/src/main/java/de/bund/digitalservice/ris/search/controller/api/NormsController.java

```diff
@@ -411,8 +424,18 @@ public ResponseEntity<String> getLegislationArticleAsHtml(
               example = "hauptteil-1_art-1")
           @PathVariable
           String articleEid,
-      @PathVariable @Schema(allowableValues = "html") String format)
+      @PathVariable @Schema(allowableValues = "html") String format,
+      @RequestHeader(
+              name = ApiConfig.Headers.GET_RESOURCES_VIA,
```

Comment:
Since this is a get operation, I would suggest to move this to a url parameter instead of a request header and make a validation on this url parameter with the enum values, we can also give it a default value if it is not set to be the non-api version. I think this would be a more compatible way with the REST get methods in my opinion.